src/webpack.js

```diff
@@ -7,6 +7,8 @@ const webpack = require('webpack');
 const WrapperPlugin = require('wrapper-webpack-plugin');
 const ZipPlugin = require('zip-webpack-plugin');
 
+const WEBPACK_DEFAULTS = new webpack.WebpackOptionsDefaulter().process({});
```

Comment:
I am having trouble finding documentation about the use of this API. Could your provide a link?

Reply:
I don't know of any documentation

Reply:
Do we know that this component is stable and intended for external consumption then? When I briefly looked at the code it seemed to be working with a lot of different things.

Reply:
I'm not sure about an official position on it's stability. If it stops working when we upgrade to a new version we should notice it pretty quickly since it's tested and we can always revert back to the hard-coded list if they break in some way that the defaults cannot be extended gracefully

Reply:
I'd honestly rather not use undocumented APIs as it makes it more difficult to understand the code here and difficult to understand why/how the code broke (should that happen). For the sake of argument, what do we lose by sticking with the hard-coded approach?